superset/config.py

```diff
@@ -153,6 +153,11 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 # default time filter in explore
 # values may be "Last day", "Last week", "<ISO date> : now", etc.
 DEFAULT_TIME_FILTER = NO_TIME_RANGE
+TIME_FILTER_SHORTCUTS = [
+    ["Current week", "DATETRUNC(DATETIME('today'), WEEK) : today"],
+    ["Current month (MTD)", "DATETRUNC(DATETIME('today'), MONTH) : today"],
+    ["Current year (YTD)", "DATETRUNC(DATETIME('today'), YEAR) : today"],
+]
```

Comment:
A minor thing, but would it make sense to define these as a map instead of an array? I will review/test in more detail later in the week.

Reply:
Thanks for the review Ville! The reseaon of using an array is to create an ordered list for the 'shortcut', e.g. week before month